src/settings/storage/node.js

```diff
@@ -30,7 +30,12 @@ export function validateStorage(settings) {
         url,
         tls,
         connectionTimeout,
-        operationTimeout
+        operationTimeout,
+        redisTlsEnabled,
+        redisTlsCaCertificates,
+        redisTlsSkipNameValidation,
+        redisTlsClientCertificate,
+        redisTlsClientKey
       } = options;
```

Comment:
Not a part of this PR: but just wondering
will it be possible to define a type for this Object so that clients using it can know what options they can use instead of coming here and checking the code?

Currently this is what I am seeing in Node SDK 10.17.0
```
  readonly storage: {
    prefix: string,
    options: Object,  <-- type here
    type: StorageType
  },

Reply:
That's a good point. We will include the type of the `options` object :)